src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -374,22 +383,25 @@ private void contributeItemsChunk(UUID centralServerId, String bibId, List<Item>
       // not possible to guess what item failed when the chunk of multiple items is being contributed
       var recordId = items.size() == 1 ? items.get(0).getId() : null;
       itemExceptionListener.logWriteError(e, recordId);
+      log.info("contributeItemsChunk:: recordId: {}, exception occurred: {}", recordId, e);
       addRecordProcessed();
     } finally {
+      log.info("contributeItemsChunk:: finally added processed records and updated stats");
```

Comment:
Not sure this is really that meaningful.

Reply:
Removed